packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js

```diff
@@ -1,33 +1,25 @@
 Terra.describeViewports('Label Value View', ['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], () => {
-  describe('when no value input is provided', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/default-label-value-view');
-    });
+  it('when no value input is provided', () => {
+    browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/default-label-value-view');
```

Comment:
as part of this please remove all '/#' from the urls, it's just an un-needed redirect at this point.

Reply:
```suggestion
    browser.url('/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/default-label-value-view');
```